cookbooks/bcpc-hadoop/recipes/oozie_pack.rb

```diff
@@ -0,0 +1,99 @@
+#
+# Recipe to include hbase jar files into Oozie
+#
+%w{
+hbase
+}.each do |p|
+  package p do
+    action :install
+  end
+end
+
+bash "/tmp/oozie.war" do
+  user "root"
+  group "root"
+  code <<-EOH
+    cp /usr/hdp/2.2.0.0-2041/oozie/oozie-server/webapps/oozie.war /tmp/oozie.war
+  EOH
+  action :run
```

Comment:
We want the chef-client run to fail if the oozie.war is not found. Since this recipe need to run after the ```oozie`` recipe this step should not fail. If it fails we have a bigger issue which need to be looked at.